setup.rb

```diff
@@ -12,6 +12,17 @@
   "2.1" => "releases/1.7"
 }
 
+# outputs system command as well as running the command
+module Kernel
+  def system_with_debug(*args)
+    puts "$ #{args.join(" ")}"
```

Comment:
What do you think about something more explicit (and less confusing with something manually from from the command line) such as "Running step: "